apps/frontend/src/components/global/upload_tabs/LoadFileList.vue

```diff
@@ -35,26 +83,97 @@
 <script lang="ts">
 import Vue from 'vue';
 import Component from 'vue-class-component';
+import EditEvaluationModal from '@/components/global/upload_tabs/EditEvaluationModal.vue'
 
+import {SnackbarModule} from '@/store/snackbar';
+import {EvaluationModule} from '@/store/evaluations'
 import {IEvaluation} from '@heimdall/interfaces';
 import {Prop} from 'vue-property-decorator';
-import {Samples} from 'aws-sdk/clients/devicefarm';
+import {Sample, Samples} from 'aws-sdk/clients/devicefarm';
```

Comment:
I'm pretty sure this is the completely wrong sample altogether (pretty sure this one is on me, sorry). This should be:

```javascript
import {Sample} from '@/utilities/sample_util';
```

Once this is changed the filtered file search function gets simplified a bit.

Reply:
That makes more sense, fixed now.